File: algorithm/src/main/java/baekjoon/p10810/Main.java
package baekjoon.p10810;

import java.util.Arrays;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int m = sc.nextInt();
        int[][] inputs = new int[m][3];

        for (int[] input : inputs) {
            input[0] = sc.nextInt();
            input[1] = sc.nextInt();
            input[2] = sc.nextInt();
        }

        for (int solve : solve(n, inputs)) {
            System.out.print(solve + " ");
        }
    }

    private static int[] solve(int n, int[][] inputs) {
        int[] ret = new int[n];
        for (int[] input : inputs) {
            Arrays.fill(ret, input[0] - 1, input[1], input[2]);
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p10810/Main.kt
package baekjoon.p10810

fun main(args: Array<String>) {
    val (n, m) = readLine()!!.split(" ").map { it.toInt() }
    val inputs = List(m) { readLine()!!.split(" ").map { it.toInt() } }
    println(solve(n, inputs).joinToString(" "))
}

private fun solve(n: Int, inputs: List<List<Int>>): Array<Int> {
    val ret = Array(n) { 0 }
    for (input in inputs) {
        ret.fill(input[2], input[0] - 1, input[1])
    }

    return ret
}